src/components/login/LoginPage.js

```diff
@@ -0,0 +1,27 @@
+import React, {Component} from 'react';
+import  PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import LoginForm from './LoginForm';
+import {login} from '../../redux/actions/auth';
+
+class LoginPage extends Component{
+
+    submit=data => this.props.login(data).then(()=> this.props.history.push("/"));
+
+    render(){
+        return(
```

Comment:
I really think it is confusing to me which component is presentational or  functional since you have `dispatched an action in a presentational component` instead we are supposed to get `data through props` from the LoginForm.I really consider you checking the differences between presentational and functional and also where to pass a `child component in a parent component`.`I think that  is not advisable to pass an action in a presentational component`
https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0